src/fabric/test/fabric2_db_misc_tests.erl

```diff
@@ -210,8 +210,8 @@ validate_doc_ids(_) ->
         meck:expect(
             config,
             get,
-            ["couchdb", "max_document_id_length", "238"],
-            "16"
+            ["couchdb", "max_document_id_length", 238],
```

Comment:
`config:get/3` should return a list (string) I believe. Also if we are testing what happens when the value is set to 16 should we still return "16"?

Reply:
See https://github.com/apache/couchdb/pull/3632/commits/468123da973d22ff9d4e91477b1f48568cde80b9.